src/homeWork7/Cat.java

```diff
@@ -0,0 +1,27 @@
+package homeWork7;
+
+public class Cat extends Animals {
+
+    private static int counterCats;
```

Comment:
Would be better to rename `counterCats` to `catsCounter` and also rename `getCounterCat` to `getCatsCounter`